account_invoice_fiscal_position_update/tests/test_inv_fiscal_pos_update.py

```diff
@@ -0,0 +1,122 @@
+# © 2014 ToDay Akretion (http://www.akretion.com)
+# @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.account.tests.account_test_classes import AccountingTestCase
+import time
+
+
+class TestProductIdChange(AccountingTestCase):
+    """Test that when an included tax is mapped by a fiscal position,
+    when position fiscal change taxes and account wil be update on
+    invoice lines.
+    """
+
+    def setUp(self):
+        super(TestProductIdChange, self).setUp()
+        self.invoice_model = self.env['account.invoice']
+        self.fiscal_position_model = self.env['account.fiscal.position']
+        self.fiscal_position_tax_model = \
+            self.env['account.fiscal.position.tax']
+        self.fiscal_position_account_model = \
+            self.env['account.fiscal.position.account']
+        self.tax_model = self.env['account.tax']
+        self.account_model = self.env['account.account']
+        self.pricelist_model = self.env['product.pricelist']
+        self.res_partner_model = self.env['res.partner']
+        self.product_tmpl_model = self.env['product.template']
+        self.product_model = self.env['product.product']
+        self.invoice_line_model = self.env['account.invoice.line']
+        self.account_user_type = self.env.ref(
+            'account.data_account_type_revenue')
+        self.account_receivable = self.env['account.account'].search(
+            [('user_type_id', '=',
+                self.env.ref('account.data_account_type_receivable').id)],
+            limit=1)
+        self.account_revenue = self.env['account.account'].search(
+            [('user_type_id', '=',
+                self.account_user_type.id)],
+            limit=1)
+
+    def test_fiscal_position_id_change(self):
+        partner = self.res_partner_model.create(dict(name="George"))
+        account_export_id = self.account_model.sudo().create({
```

Comment:
account_export_id = self.account_model.sudo().search(
            [('code', '=', '710000')], limit=1
        )
        if not account_export_id:
            account_export_id = self.account_model.sudo().create({
                'code': "710000",
                'name': "customer export account",
                'user_type_id': self.account_user_type.id,
                'reconcile': True,
            })